files/cf-exclude-include.json

```diff
@@ -51,14 +51,13 @@
   ],
   "modpacks": {
     "all-of-fabric-6": {
-      "forceIncludes": [
-        "revelationary"
-      ]
+      "forceIncludes": ["revelationary"]
     },
     "valhelsia-5": {
-      "excludes": [
-        "modernfix"
-      ]

```

Comment:
It would be helpful if you could revert these extra diffs.

Reply:
Yes my bad, we probably use a different formater. Pushing a fix. 
Thanks for the awesome work on this dockerfile btw, loving it

Reply:
Also, I wonder why is `just-enough-resources-jer` in the `globalExcludes` at first ?

Reply:
It looks like I needed to exclude it for MC Eternal to startup correctly

https://github.com/itzg/docker-minecraft-server/pull/2143

I may have confused it with the "too many items" series of mods by putting it in the global excludes. Is it actually valid to run that mod on the server side?

Reply:
Yes this mod is valid on the server side, and even needed in some cases it seems